example/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
```

Comment:
Suggest to create a sub dir named `simplestorage` in example, and then move all files into it. **Remember update the  links of these files in  `README.md`.**

Reply:
Check it again. Once this pr no problem and merged, update docs later.